src/Block/Service/MenuBlockService.php

```diff
@@ -130,100 +105,41 @@ public function getMetadata(): MetadataInterface
         ]);
     }
 
-    /**
-     * @return array<array{string, class-string<FormTypeInterface>, array<string, mixed>}>
-     */
-    private function getFormSettingsKeys(): array
+    protected function getFormSettingsKeys(): array
     {
         $choiceOptions = [
             'required' => false,
-            'label' => 'form.label_url',
```

Comment:
form.label_url was a bad copy paste, since it display "URL" when it's was the menu name instead.

The fact that nobody opened an issue/PR about it would means that nobody use this class.

Reply:
I start wondering if the MenuBlockService is use by someone then,

If not, we could follow the same strategy than https://github.com/sonata-project/SonataBlockBundle/issues/1109 and deprecate the MenuBlock and move all this code to the SeoBundle.
Everything would be in the BaseBreadcrumbService and this would solve all the issues...

Reply:
I would say let's focus on the BC Break fix, I would not touch the topic of deprecating all of this.